src/web/client/src/App.tsx

```diff
@@ -44,6 +47,13 @@ class App extends Component<Props, State> {
         console.log(this.state.isLoggedIn);
       })
       .catch((e) => console.error(e));
+    fetch("/auth/isAdmin")
+      .then((res) => res.json())
+      .then((isAdmin) => {
+        this.setState({ isAdmin: isAdmin });
+        console.log(this.state.isAdmin);
+      })
+      .catch((e) => console.error(e));
```

Comment:
Wouldn't it be better to do this fetch call inside the other fetch, and call it only when `isLoggedIn === true`

Reply:
yes